src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,387 @@
+| SliceThickness              | OPTIONAL              | [number]                               | Slice thickness of the tissue sample in the unit micrometers (`"um"`) (for example: `5`).                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| TissueDeformationScaling    | OPTIONAL              | [number][]                             | Estimated deformation of the tissue, given as a percentage of the original tissue size (for examples: for a shrinkage of 3%, the value is `97`; and for an expansion of 100%, the value is `200`).                                                                                                                                                                                                                                                                                                                         |
+| SampleExtractionProtocol    | OPTIONAL              | [string][] or [URI][uri]               | Description of the sample extraction protocol or URI (for example from [protocols.io](https://www.protocols.io/)).                                                                                                                                                                                                                                                                                                                                                                                                         |
+| SampleExtractionInstitution | OPTIONAL              | [string][]                             | The name of the institution in charge of the extraction of the sample, if different from the institution in charge of the equipment that produced the image.                                                                                                                                                                                                                                                                                                                                                               |
+
+#### Chunk transformations
+
```

Comment:
If there are data chunks and the chunk information is not required, how should these chunks be interpreted?

Reply:
These chunks transformations fields are the only place to indicated how the chunks are ordered. 
The idea is to provide a field were the information could be stored if needed, but not mandatory, as it is not always relevant or available, for examples:
1) We have an example or "unordered" chunks in `Figure 1c)`, where there is no explicit spatial relation between the chunks.
2) When converting "old" dataset to BIDS, the spatial relation between "ordered" chunks may not be known.

Reply:
Then maybe there could be an `unordered` option or similar. 
Is this order relevant for the interpretation of the images? I would feel so (but I'm not expert on microscopy). If this is the case, we may want to avoid this info get lost in the process and forget how it was done some years later. Therefore we may want to encourage reporting this metadata.

Reply:
> Is this order relevant for the interpretation of the images? I would feel so (but I'm not expert on microscopy). If this is the case, we may want to avoid this info get lost in the process and forget how it was done some years later. Therefore we may want to encourage reporting this metadata.

The information is often relevant but not always depending on the case.
But I agree that we should encourage this metadata to be used when relevant! As mentioned in your other comments, I'll clarify in the chunk-index section how to track the position of the chunks and I'll make theses fields RECOMMENDED instead of OPTIONAL.

Reply:
Sounds good. Or even REQUIRED in case of having chunks

Reply:
There's probably a use case where the chunks don't have an associated transform, e.g:
* Chunks are uploaded but their relationship is not yet computed.
* Chunks' relationships to each other are not known or not relevant (for instance, if the microscope has taken unrelated patches of the same tissue)
I'd make it RECOMMENDED.

Reply:
I made the matrix RECOMMENDED as it covers most cases i.e. unordered (no chunk transform) or ordered (with chunk transforms if available).
I would prefer not to add another field just to indicate if it's ordered or not.
The descriptions of the fields also already include examples so I did not add other examples.
Let me know if the corrected version works for you. Thanks!

Reply:
I made the matrix RECOMMENDED as it covers most cases i.e. unordered (no chunk transform) or ordered (with chunk transforms if available).
I would prefer not to add another field just to indicate if it's ordered or not.
The descriptions of the fields also already include examples so I did not add other examples.
Please let me know your thoughts.

Reply:
Ok. Sounds good. Yet, I wonder, if we have ordered chunks and this field is missing, could we still proceed with the data analysis? or having this information missing will prevent from future analysis giving problems in the interpretation?
(We could have `None` in case of not having the info, for example, if this helps)

Reply:
There may be cases where you have ordered chunks and no chunk transform - if the data is implicitly tiled in a known order so that one tile starts where the last ended. The data would be analyzable in that case. I would definitely recommend that dataset constructors include the transform even in this case as documentation of the tiling layout.

There's another use case where it's up to the user of the data to calculate the offsets of the chunks relative to each other using an alignment algorithm. For our data, you can't count that the location of a chunk is correct based on the purported stage coordinates - we have had runs where the stage has become slightly tilted or twisted and a computational alignment must be done. Arguably, the resulting transform is part of the analysis, not the raw data?

So at least two cases where the reconstruction is possible without the matrix.

Reply:
> Ok. Sounds good. Yet, I wonder, if we have ordered chunks and this field is missing, could we still proceed with the data analysis? or having this information missing will prevent from future analysis giving problems in the interpretation?

My understanding is "it depends on the case".
In some cases, the order of the chunks is important for image analysis and should be preserved to reconstruct the image/volume so we recommend the use of the matrix.
I think there are others cases where the chunks are only different fields-of-view of the same sample and, even if they appear "ordered", it may not be necessary to keep track of their exact order.

> (We could have `None` in case of not having the info, for example, if this helps)

I'm not sure I understand what you mean here. Is it that the ChunkTransformationMatrix could be either `the matrix` if available or the value `None` if not?

Reply:
Not sure if this is possible technically, but yes, I was thinking of something like this

Reply:
Not sure if this is possible technically, but yes, I was thinking of something like this

Reply:
I made a few research.
There are some REQUIRED fields in the spec that accept explicitly a specific value (ex: a number) or `n/a`.
There is an interesting discussion going on about this [here](https://github.com/bids-standard/bids-specification/issues/876).

In our case, we recommend `ChunkTransformationMatrix` when the information is available. So if the field is not there, I would interpret that as not applicable/not available.
I'm sure what the benefit of having the field REQUIRED with `n/a` is in that case?

Reply:
Hi @guiomar, as mentioned above, I would be more in favor of keeping the `ChunkTransformationMatrix` RECOMMENDED only. Please let me know your thoughts. Thank you!